index.js

```diff
@@ -167,10 +168,101 @@ module.exports = ({ app }) => {
   }
 }
 
-function handleOneCommit (pr, dcoFailed) {
-  return `You only have one commit incorrectly signed off! To fix, first ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally). Next, head to your local branch and run: \n\`\`\`bash\ngit commit --amend --no-edit --signoff\n\`\`\`\nNow your commits will have your sign off. Next run \n\`\`\`bash\ngit push --force-with-lease origin ${pr.head.ref}\n\`\`\``
-}
+function handleCommits (pr, commitLength, dcoFailed, allowRemediationCommits) {
+  let returnMessage = ''
+
+  if (dcoFailed.length === 1) {
+    returnMessage = 'There is one commit incorrectly signed off.  This means that the author of this commit failed to include a Signed-off-by line in the commit message.\n\n'
+  } else {
+    returnMessage = `There are ${dcoFailed.length} commits incorrectly signed off.  This means that the author(s) of these commits failed to include a Signed-off-by line in their commit message.\n\n`
+  }
+
+  returnMessage = returnMessage + `To avoid having PRs blocked in the future, always include \`Signed-off-by: Author Name <authoremail@example.com>\` in *every* commit message. You can also do this automatically by using the -s flag (i.e., \`git commit -s\`).
+
+Here is how to fix the problem so that this code can be merged.\n\n`
+
+  let rebaseWarning = ''
+
+  if (allowRemediationCommits.individual || allowRemediationCommits.thirdparty) {
```

Comment:
When testing the feature, I never made it into this if statement - here's the yaml file i used: https://github.com/hiimbex/Test/blob/master/.github/dco.yaml & the test PR i created: https://github.com/hiimbex/Test/pull/4 - totally could be something wrong in how I'm testing, so let me know, but this scenario was my interpretation from the docs.

Reply:
Hmm, this is odd. I've been staring at this and must be missing something obvious... it was working earlier in development and appears to have stopped now. It's like it isn't reading the `dco.yml` file correctly and is always defaulting to the `false` values for both. Does anything jump out around line 20?

I guess this wasn't surfaced in the `dco.js` tests because those two variables are defined independent of what's in `dco.yml` for each test. I'm coming up against the limits of my knowledge of jest here, and suspect this would have been detected had there been appropriate snapshot tests that include permutations of `dco.js`. I'll admit I don't know how to do this aside from literally changing the default value in `index.js`, do you have any suggestions?